users_controller.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, make_response
+from flask_restx import Api, Resource, fields
+
+app = Flask(__name__)
+api = Api(
+    app,
+    version="1.0",
+    title="barax",
+    description="REST API for our guild management application.",
+)
+
+
+@app.route("/users")
+def index():
+    all_users = {"users": users}
+    return all_users
+
+
+@app.route("/users", methods=["POST"])
+def create():
+    user = {
+        "name": request.json["name"],
+        "email": request.json["email"],
+        "password": request.json["password"],
+    }
+    users.append(user)
+    return make_response("", 204)
+
+
+@app.route("/users/<int:user_id>", methods=["DELETE"])
+def destory(user_id):
+    for i in range(len(users)):
+        if users[i]["id"] == user_id:
+            del users[i]
+            break
+    return make_response("", 204)
```

Comment:
We should be able to return a separate response indicating that a user with that id was not found.